website/blog/2019-08-31-dandelion-overview.md

```diff
@@ -1,68 +1,75 @@
 ---
-title: 🌼 Dandelion = 👹 Moloch + 🦅 Aragon
+title: 🌼 Dandelion = 👹 MolochDAO + 🦅 Aragon
 author: 1Hive team
 authorURL: https://github.com/1hive
 ---
 
-1Hive recently received an Aragon Nest [grant](https://github.com/aragon/nest/issues/168) to build Dandelion Orgs, a set of Aragon applications and an organization template for deploying an Aragon DAO that is functionally similar to [Moloch](https://molochdao.com), but with the notable advantage of modularity and extensibility provided by the Aragon platform and its growing library of applications.
+1Hive recently received an Aragon Nest [grant](https://github.com/aragon/nest/issues/168) to build Dandelion Orgs, a set of Aragon applications and an organization template for deploying an Aragon DAO that is functionally similar to [MolochDAO](https://MolochDAOdao.com), but with the notable advantage of modularity and extensibility provided by the Aragon platform and its growing library of applications.
```

Comment:
I think you updated the URL from the correct URL to an incorrect URL here?

Reply:
Good catch. Thanks!